tensorflow/lite/kernels/neg_test.cc

```diff
@@ -75,5 +103,61 @@ TEST(NegOpModel, NegInt64) {
   EXPECT_THAT(m.GetOutput<int64_t>(), ElementsAreArray({2, 1, 0, -1, -2, -3}));
 }
 
+class NegOpQuantizedModel : public NegOpModel {
+ public:
+  NegOpQuantizedModel(const TensorData &input, const TensorData &output)
```

Comment:
Please place the & on the type, not the name. `const TensorData& input`.

Reply:
Addressed the requested change in [#29eaf8a](https://github.com/MCW-Dev/tensorflow/pull/49/commits/29eaf8ad5b0518a4b04d13c1fa7d5e3cc040b195)